src/app/components/loaders/src20-tokens-loader.tsx

```diff
@@ -9,8 +13,12 @@ export interface Src20TokenAssetDetails {
 interface Src20TokensLoaderProps {
   address: string;
   children(tokens: Src20TokenAssetDetails[]): React.ReactNode;
+  filter?: AssetFilter;
 }
-export function Src20TokensLoader({ address, children }: Src20TokensLoaderProps) {
+export function Src20TokensLoader({ address, children, filter = 'all' }: Src20TokensLoaderProps) {
   const { data: tokens = [] } = useSrc20TokensByAddress(address);
-  return children(tokens);
+  const accountIndex = useCurrentAccountIndex();
```

Comment:
This code seems almost the same as `Brc20TokensLoader`, do you think we could share it a bit better?